platform/ios/src/MGLLocationManager.m

```diff
@@ -1,9 +1,9 @@
 #import "MGLLocationManager.h"
 #import <UIKit/UIKit.h>
 
-static const NSTimeInterval MGLLocationManagerFiveMinuteTimeInterval = 300.0;
-static const NSTimeInterval MGLLocationManagerFiveSecondTimeInterval = 5.0;
-static const CLLocationDistance MGLLocationManagerRegionRadiusLocationDistance = 300.0;
+static const NSTimeInterval MGLLocationManagerHibernationTimeout = 300.0;
+static const NSTimeInterval MGLLocationManagerHibernationPollInterval = 5.0;
+static const CLLocationDistance MGLLocationManagerHibernationRadius = 300.0;
```

Comment:
@1ec5 these will probably change again in the future but hopefully this is a bit clearer. I'd like to keep them as constants, for now, since this class might someday evolve be something more configurable and be used by other parts of the SDK or even by developers who have similar battery sensitive location manager needs.